src/Database/Table/Selection.php

```diff
@@ -334,6 +334,35 @@ public function where($condition, ...$params)
 		return $this;
 	}
 
+	/**
+	 * Adds where condition using the OR operator between parameters.
+	 * More calls appends with AND.
+	 * @param  array array('column1' => 1, 'column2 > ?' => 2, 'full condition')
+	 * @return self
+	 */
+	public function whereOr(array $parameters)
+	{
+		if (count($parameters) >= 2) {
```

Comment:
If you invert this condition, you can simply avoid putting whole method body into one large condition.